src/core/filesystem.cc

```diff
@@ -204,6 +214,21 @@ LocalFileSystem::ReadTextFile(const std::string& path, std::string* contents)
   return Status::Success;
 }
 
+Status
+LocalFileSystem::DownloadFileFolder(
+    const std::string& path, std::string* local_path)
+{
+  *local_path = path;
```

Comment:
Put a comment "For local file system we don't actually need to download the folder... just use it in place"